hw/ip/flash_ctrl/data/flash_ctrl.hjson

```diff
@@ -10,7 +10,8 @@
   reset_primary: "rst_ni",
   other_reset_list: [ "rst_otp_ni" ],
   bus_interfaces: [
-    { protocol: "tlul", direction: "device" }
+    { protocol: "tlul", direction: "device", name: "core" }
+    { protocol: "tlul", direction: "device", name: "prim" }
```

Comment:
do we have plan to support arbitrary clock domain for each tlul interface?

Reply:
i think at the moment that's not there.. so it would be up to the surrounding logic to place the regfile in the right domain and handle any async crossing.  It would be great if we had it though :)

Reply:
thanks for the explanation. So, this is basically logical partition at this time (logical and + separate address)

Reply:
yeah that's right.